app/assets/stylesheets/application.scss

```diff
@@ -15,10 +15,48 @@
  *= require foundation_and_overrides
  */
 
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700,400italic,300italic);
+
  /************** BASIC STYLING *****************************/
 
 #content {
-  background-color: #DADADA;
+  background-color: #FFF;
+  margin-bottom: 6rem;
+  margin-top: 6rem;
+}
+
+#background {
+  display: flex;
+  flex-flow: column;
+  height: 100%;
+  background: url("http://freephotos.atguru.in/hdphotos/forest-landscape-wallpaper/forest-sun-wallpaper-landscape-nature-wallpaper-2560-1600-widescreen-1080.jpg") no-repeat center center fixed;
```

Comment:
URL `http://freephotos.atguru.in/hdphotos/forest-landscape-wallpaper/forest-sun-wallpaper-landscape-nature-wallpaper-2560-1600-widescreen-1080.jpg` should not contain protocol or domain